doc/man/Makefile.am

```diff
@@ -1,27 +1,27 @@
 dist_man1_MANS=
 
 if BUILD_BITCOIND
-  dist_man1_MANS+=vertcoind.1
+  dist_man1_MANS+=bitcoind.1
 endif
 
 if ENABLE_QT
-  dist_man1_MANS+=vertcoin-qt.1
+  dist_man1_MANS+=bitcoin-qt.1
 endif
 
 if BUILD_BITCOIN_CLI
-  dist_man1_MANS+=vertcoin-cli.1
+  dist_man1_MANS+=bitcoin-cli.1
 endif
 
 if BUILD_BITCOIN_TX
-  dist_man1_MANS+=vertcoin-tx.1
+  dist_man1_MANS+=bitcoin-tx.1
 endif
 
 if BUILD_BITCOIN_UTIL
-  dist_man1_MANS+=vertcoin-util.1
+  dist_man1_MANS+=bitcoin-util.1
 endif
 
 if ENABLE_WALLET
 if BUILD_BITCOIN_WALLET
-  dist_man1_MANS+=vertcoin-wallet.1
+  dist_man1_MANS+=bitcoin-wallet.1
```

Comment:
What is this commit?

Reply:
I corrected the commit by removing the manpage changes from 'vtc branding'.  Building the manpages needs to be a part of the release process.